db_api_etl/nci_api_db.sql

```diff
@@ -205,6 +205,20 @@ CREATE TABLE parents (
   level int
 );
 
+drop table if exists trial_prior_therapies;
+CREATE TABLE trial_prior_therapies(
+  idx SERIAL PRIMARY KEY,
+  nct_id VARCHAR(100),
+  nci_thesaurus_concept_id VARCHAR(25),
+  eligibility_criterion TEXT,
+  inclusion_indicator TEXT,
+  name TEXT,
+  CONSTRAINT trial_prior_therapies_trial_fk FOREIGN KEY (nct_id) REFERENCES trials(nct_id),
+  CONSTRAINT trial_prior_therapies_ncit_fk FOREIGN KEY (nci_thesaurus_concept_id) REFERENCES ncit(code)
```

Comment:
Hmm, if we see the same sort of behavior as with biomarkers here -- the abstractors will have access to NCIt concept codes that have not been released by EVS - it would still come over in the this data, but would likely not have a name filled in, and would be in our NCIT table.  So I think comment out the  trial_prior_therapies_ncit_fk constraint in line 217 because we don't want to error out in that case.

Reply:
Thanks for the background.  Done.